### PR TITLE
feat: add showIndicators and showNavigators properties to carousel co…

### PR DESCRIPTION
**Pull request by ulasturann**
…mponent.

### Feature Requests
Add showIndicators and showNavigators props to Carousel Component.

### CODE REVIEW

File: components/doc/carousel/index.js [hunk 1]
```diff
@@ -549,6 +549,24 @@ const responsiveOptions = [
                                     <td>false</td>
                                     <td>Defines if scrolling would be infinite.</td>
                                 </tr>
+                                <tr>
```
Comment (melloware) [inline components/doc/carousel/index.js:552]: Also please update `\api-generator\components\carousel.js`

**Review by melloware — CHANGES_REQUESTED** [review]

File: components/doc/carousel/index.js (hunk 1 above)
Comment (ulasturann) [reply components/doc/carousel/index.js:552]: > Also please update `\api-generator\components\carousel.js`
Thank you for your kind reminder @melloware. I added the necessary attributes to the file as you specified.

**Comment by mertsincan** [comment]
@ulasturann, Thanks a lot for your contribution!

**Comment by ulasturann** [comment]
> @ulasturann, Thanks a lot for your contribution!

Thanks for your help @mertsincan .

**Pull request merged**